.github/workflows/build.yml

```diff
@@ -49,6 +49,10 @@ jobs:
         run: cmake --build build --target tests
       - name: Run tests
         run: build/tests/tests
+      - name: Display help
```

Comment:
Note: I am not sure if this is the right place. We might rather include it in the previous stage, but I think the tests currently do not support executing the lmms executable, and are not made for that task.

Reply:
I will change this to use the `make install`-installed lmms instead.